source/_components/camect.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: "Camect"
+description: "How to integrate Camect into Home Assistant."
+date: 2019-03-25 15:28
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: camect.png
+ha_category:
+  - Hub
+  - Camera
+ha_release: 0.91
+ha_iot_class: Local Polling
+redirect_from:
+  - /components/camera.camect/
+---
+
+The Camect component sets up the integration with your [Camect](https://www.camect.com) instance.
+Let you watch your camera streams using WebRTC. It's live stream, not image snapshots.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Camera](#camera)
+
+## {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+camect:
```

Comment:
Add the port variable because it is required follow the python code: https://github.com/home-assistant/home-assistant/pull/22446/files#diff-2bf66a1a1a1fd28f46737f127f5df1e7R35

Reply:
done